packages/application-shell/src/components/application-shell/application-shell.js

```diff
@@ -438,17 +429,19 @@ export default class ApplicationShell extends React.Component {
                           ),
                         }),
                   };
-                  // TODO: when we remove the login routes from the AppShell, this component
-                  // should do a redirect/hard page reload to the login route.
-                  // For development, we should only do a redirect to the staging domain, as the
-                  // login screen will not be part of the AppShell anymore.
-                  return (
-                    <Redirect
-                      to={{
-                        pathname: '/login',
-                        query: searchQuery,
-                      }}
-                    />
+
+                  /**
+                   * NOTE:
+                   *   Given the application runs in production we redirect to that host's login application.
+                   *   Given the application is in development mode (locally) we redirect to the application on staging.
+                   */
+                  const frontendHost =
+                    process.env.NODE_ENV === 'production'
+                      ? ''
+                      : this.props.environment.frontendHost;
```

Comment:
I don't think we can use this. For local development this is `localhost:3001`. We should probably hardcode the staging domain here, or we add a new value e.g. `loginUrl` or `authUrl` in `env.json`

Reply:
Can you elaborate why `frontendHost` is not working? Adding `loginUrl` sounds reasonable but I don't follow how that is not just `${frontendHost}/login`.

Reply:
Well, in development `frontendHost` points to `localhost:3001`. So when you go to `/login`, we end up in an infinite loop because the page keeps reloading as there is no "login service" running.

Instead, it should point to `https://mc.commercetools.com/login`, right? (or our staging domain, which I won't write down).

Reply:
Understood. Thanks. Then I will add a new env variable called `authUrl` to have it be "augmented" to `/login` and `/logout`.